pkgs/applications/science/electronics/openems/default.nix

```
@@ -0,0 +1,75 @@
+{ stdenv
```

since lib will be used a lot
```suggestion
{ stdenv
, lib
```